Conventions.md

```diff
@@ -3,7 +3,7 @@
 ## Collaboration model
 
 The `master` branch should be left untouched and we can contribute to its changes from separate branches.
-
+a
```

Comment:
Thanks a lot for the change, I think this `a` is here by accident, correct?

Reply:
You're right !